src/common/libschedutil/ops.h

```diff
@@ -54,13 +60,17 @@ struct schedutil_ops {
                    const flux_msg_t *msg,
                    void *arg);
 
+    int cancel_flags;
+
     /* The job manager wants to change the priority one or more jobs.
      * This callback is not required and can be set to NULL if the
      * caller wishes to ignore these messages.
      */
     void (*prioritize)(flux_t *h,
                        const flux_msg_t *msg,
                        void *arg);
+
+    int prioritize_flags;
```

Comment:
The individual flags variables here in the ops structure seem like overkill.  What about just adding a single flags argument to `schedutil_create()`?